File: c/1011.c

// https://www.urionlinejudge.com.br/judge/pt/problems/view/1011

#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    double volume, pi, r;
    pi = 3.14159;

    scanf("%lf", &r);
    volume = (4.0/3) * pi * pow(r,3);

    printf("VOLUME = %.3lf\n", volume);

    return 0;
}

File: haskell/1011.hs

-- https://www.urionlinejudge.com.br/judge/pt/problems/view/1011

import Text.Printf (printf)

main :: IO ()
main = do r <- readLn :: IO Double
          printf "VOLUME = %.3f\n" ((((4.0/3) * 3.14159)*(r^3)) :: Double)